src/illuminatio/cleaner.py

```diff
@@ -1,3 +1,6 @@
+"""
+Contains the clean functionality
```

Comment:
`Contains the routines to completely clean up all resources generated by illuminatio`